wallet/walletrpc.c

```diff
@@ -369,14 +369,36 @@ static void json_listfunds(struct command *cmd, const char *buffer UNUSED,
 	struct peer *p;
 	struct utxo **utxos =
 	    wallet_get_utxos(cmd, cmd->ld->wallet, output_state_available);
+	char* out;
+	struct ripemd160 h160;
+	struct pubkey funding_pubkey;
 	json_object_start(response, NULL);
 	json_array_start(response, "outputs");
 	for (size_t i = 0; i < tal_count(utxos); i++) {
 		json_object_start(response, NULL);
 		json_add_txid(response, "txid", &utxos[i]->txid);
 		json_add_num(response, "output", utxos[i]->outnum);
 		json_add_u64(response, "value", utxos[i]->amount);
-
+		// @closed_info is for outputs that are not yet claimable
```

Comment:
`closed_info` misspelled, also, we use `/* */` comments.

Reply:
Fixed this quickly myself :-)